README.md

```diff
@@ -93,7 +93,30 @@ btcdeb>
 
 ### Signature checking
 
-In order to run an OP_CHECKSIG command, the debugger needs to know about the transaction being checked, since it creates the signature hash from the transaction content. You can pass the transaction to `btcdeb` when you run it, using the `--tx=[amount1[,amount2[,...]]]:[hexdata]`, where `[amountN]` is the amount of the inputs of the transaction, and `[hexdata]` is the hexadecimal representation of the entire transaction (not just the transaction ID).
+In order to run an OP_CHECKSIG command, the debugger needs to know about the transaction being checked, since it creates the signature hash from the transaction content. You can pass the transaction to `btcdeb` when you run it, using the `--tx=amount1,amount2:hexdata`, where `amountN` is the amount of the inputs of the transaction, and `[hexdata]` is the hexadecimal representation of the entire transaction (not just the transaction ID). For example, to verify transaction ID c2fdfbcbef9acb6107eb5d18c172f234ee694254be1128d29b85b80b9bad9b3a, the following will produce an output of TRUE.
```

Comment:
Could you also change `[hexdata]` to `hexdata` here since you changed the amount ones?

Reply:
made requested change in commit 739b1fb